lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -792,9 +795,11 @@ DefConstructor('\usepackage OptionalSemiverbatim Semiverbatim []',
   afterDigest  => sub { my ($stomach, $whatsit) = @_;
     my $options  = $whatsit->getArg(1);
     my $packages = $whatsit->getArg(2);
-    my @pkgs     = grep { $_ } grep { !/^\s*%/ } split(/\s*,\s*/, ToString($packages));
     $options = [($options ? split(/\s*,\s*/, (ToString($options))) : ())];
-    map { RequirePackage($_, options => $options) } @pkgs;
+    for my $pkg (split(',', ToString($packages))) {
+      $pkg =~ s/\s+//g;
+      next if !$pkg || $pkg =~ /^%/;
+      RequirePackage($pkg, options => $options); }
     return });
```

Comment:
Very strange.  With https://corpora.mathweb.org/entry/import/2447837 I'm having suddenly most packages not load (even though they end up in PI's), so that the `\usetikzlibrary` line fails undefined.  It's so nonsensical, I can't pin down what's happening. Seems like a Perl bug, or even a memory error (I even rebooted).  Do you see anything odd?

Reply:
I think I just found a fix for this, but it is so sinister! Found it independently too. PR incoming...

Reply:
Linking to PR to keep the issue connected: #1411